plugin/autopath/README.md

```diff
@@ -50,18 +50,19 @@ autopath @kubernetes
 
 Use the search path dynamically retrieved from the *kubernetes* plugin.
 
-## Known Issues
+## Bugs
 
-In Kubernetes, *autopath* can derive the wrong namespace of a client Pod (and therefore wrong search path)
-in the following case.  To properly build the search path of a client *autopath* needs to
-know the namespace of the a Pod making a DNS request. To do this, it relies on the
-*kubernetes* plugin's Pod cache to resolve the client's IP address to a Pod.  The Pod cache is maintained by
-an API watch on Pods.  When Pod IP assignments change, the Kubernetes API notifies CoreDNS via the API watch.
+In Kubernetes, *autopath* can derive the wrong namespace of a client Pod (and therefore wrong search
+path) in the following case. To properly build the search path of a client *autopath* needs to know
+the namespace of the a Pod making a DNS request. To do this, it relies on the *kubernetes* plugin's
+Pod cache to resolve the client's IP address to a Pod. The Pod cache is maintained by an API watch
+on Pods. When Pod IP assignments change, the Kubernetes API notifies CoreDNS via the API watch.
 However, that notification is not instantaneous. In the case that a Pod is deleted, and it's IP is
-immediately provisioned to a Pod in another namespace, and that new Pod make a DNS lookup *before* the API watch
-can notify CoreDNS of the change, *autopath* will resolve the IP to the previous Pod's namespace.
+immediately provisioned to a Pod in another namespace, and that new Pod make a DNS lookup *before*
+the API watch can notify CoreDNS of the change, *autopath* will resolve the IP to the previous Pod's
+namespace.
 
 In Kubernetes, *autopath* is not compatible with Pods running from Windows nodes.
```

Comment:
Maybe move this out of the Bugs section... It's a limitation of k8s/windows, which doesn't support a domain search list.  Or perhaps just remove it.

Reply:
maybe indeed just remove it. On that bugs section if we only get NXDOMAIN we may be able to return a non-nonsensical answer that isn't NXDOMAIN to still short cut the resolution process; not sure what the client side implication are though

Reply:
The original incarnation of autopath had this option - to return another response to make the client stop searching. IIRC, SERVFAIL worked in my tests.  But it was an option, since I was unsure how various client resolvers, or applications would handle various negative responses.  It's probably too fiddly of a detail to clutter the config with. The failure mode is just "more work", not "wrong answer".